docker/adamant_env.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set +e
+
+if ! command -v docker &> /dev/null
+then
+  if command -v podman &> /dev/null
+  then
+    function docker() {
+      podman $@
+    }
+  else
+    echo "Neither docker nor podman found!!!"
+    exit 1
+  fi
+fi
+
+DOCKER_CONTAINER_NAME="adamant_container"
+DOCKER_IMAGE_NAME="ghcr.io/lasp/adamant:latest"
+DOCKER_COMPOSE_COMMAND="docker compose"
```

Comment:
I am thinking we should still grab these from docker_config.sh. We shouldn't expect the user to ever look at this file, but may want to modify docker_config.sh.

Reply:
Addressed in [37ca6e0](https://github.com/lasp/adamant/pull/50/commits/37ca6e0595b321efcb7d41225d9391152126db30).